src/vm/CMakeLists.txt

```diff
@@ -264,6 +264,9 @@ set(VM_SOURCES_WKS
     weakreferencenative.cpp
 )
 
+list(APPEND VM_SOURCES_WKS 
```

Comment:
Can you please move this source into the previous list of sources (keeping the list alphabetically ordered)